README.md

```diff
@@ -3,3 +3,8 @@
 # TIMDEX UI
 
 A discovery interface backed by [the TIMDEX API](https://github.com/MITLibraries/timdex).
+
+## Optional Environment Variables
+
+- `TIMDEX_V1`: value to override the default url for the TIMDEX API.
```

Comment:
I'd like us to consider using something like `TIMDEX_BASE_URL` or something instead of `TIMDEX_V1` as it'll get confusing to use set the v2 base in that ENV :)

Reply:
Yeah, I wasn't sure if there was going to be a need for us to have both APIs in env simultaneously. If we don't think that's the case, then yeah something generic like `TIMDEX_BASE_URL` makes more sense.

Reply:
That's a good point. I'm not sure we've decided if this UI will be v1/v2. I think we probably want to just have it be v2 or we triple our work and just delete it all in a few months.